feedthefox/settings.py

```diff
@@ -172,6 +173,8 @@
 ACCOUNT_AUTHENTICATION_METHOD = 'email'
 ACCOUNT_LOGOUT_REDIRECT_URL = '/'
 ACCOUNT_DEFAULT_HTTP_PROTOCOL = 'https'
+if DEBUG:
```

Comment:
This is not a really good idea because ``DEBUG`` might be enabled in our dev deployment that uses ``https``. I suggest we add an extra configuration entry that default to the value of ``DEBUG`` but can be overwritten.